File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.py
def solution(n):
    answer = 0
    
    for i in range(2, n+1):
        prime = True;
        
        for x in range(2, int(i ** 0.5) + 1):
            if i % x ==0:
                prime = False
                break
        if prime:
            answer+= 1
    
    return answer

File: 프로그래머스/1/12921. 소수 찾기/소수 찾기.cpp
#include <stdio.h>

int solution(int n) {
    int answer = 0;

    for(int i = 2; i <= n; i++) {
        int prime = 1;

        for(int x = 2; x * x <= i; x++) {
            if(i % x == 0) {
                prime = 0;
                break;
            }
        }

        if(prime) {
            answer++;
        }
    }

    return answer;
}